testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/broker/SocialLoginTest.java

```diff
@@ -195,6 +197,10 @@ private void setTestProvider(Provider provider) {
         log.infof("added '%s' identity provider", provider.id());
         currentTestProvider = provider;
         currentSocialLoginPage = Graphene.createPageFragment(currentTestProvider.pageObjectClazz(), driver.findElement(By.tagName("html")));
+
+        if(provider == OPENSHIFT4_BOXED) {
```

Comment:
This should consider even the normal `OPENSHIFT4` provider.

Reply:
done